modules/rgbd/misc/python/test/test_rgbd.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Python 2/3 compatibility
+from __future__ import print_function
+
+import os, numpy
+
+import cv2 as cv
+
+from tests_common import NewOpenCVTests
+
+class rgbd_test(NewOpenCVTests):
+
+   def test_computeRgbdPlane(self):
+
+      depth_image = self.get_sample('/cv/rgbd/depth.png', cv.IMREAD_ANYDEPTH)
+      if depth_image is None:
+         raise unittest.SkipTest("Missing files with test data")
```

Comment:
please use indentation with 4 spaces

Reply:
I think my VSCode settings were incorrect and indentation size was reset to 3. Corrected now.